api/logs/views.py

```diff
@@ -97,6 +117,7 @@ class LogNodeList(JSONAPIBaseView, generics.ListAPIView, ODMFilterMixin):
 
     def get_queryset(self):
         log = self.get_log()
+        self.check_log_permission(log)
```

Comment:
Yes, this method throws a 401 or 403 if the user doesn't have permission.  Checking permissions on logs are tricky, because whether or not the user can view the log is not stored on the log itself, but rather, if the user can view the node the log pertains to.